packages/components/iframe.tsx

```diff
@@ -24,29 +25,90 @@ const noProxyScript = `
   }
 `;
 
+/**
+ * The iframe type definitions.
+ */
 interface Props {
+  /**
```

Comment:
What about extending the types of an iframe here as we do with the [Image props](https://github.com/frontity/frontity/blob/379a44761423b4e47466359bbfe598615a869dcb/packages/components/image.tsx#L8):

```suggestion
 */
interface Props extends React.IframeHTMLAttributes<HTMLIframeElement> {
  /**
```

(I haven't tested it)

Reply:
Tried and it showed a couple of TS errors and didn't want to go deep into this as it's unrelated. But it totally make sense to do this 👍

Reply:
Okey, no problem Nícholas 🙂 👍